ddtrace/contrib/botocore/patch.py

```diff
@@ -161,10 +161,9 @@ def inject_trace_to_eventbridge_detail(params, span):
 
```

Comment:
?

Reply:
This file shouldn't be there. I'll revert this change